docker/ngc_overview_hugectr.md

```diff
@@ -0,0 +1,101 @@
+
+```
+pip install jupyterlab
+```
+
+Finally start the jupyter-lab server:
+
+```
+jupyter-lab --allow-root --ip='0.0.0.0' --NotebookApp.token=''
+```
+
+Now you can use any browser to access the jupyter-lab server, via :8888
+Once in the server, navigate to the /nvtabular/ directory and explore the code base or try out some of the examples.
+Within the container is the codebase, along with all of our dependencies, particularly RAPIDS Dask-cuDF. The easiest way to get started is to simply launch the container above and explore the examples within.
+
+### Other NVIDIA Merlin containers
+
+Merlin containers are available in the NVIDIA container repository at the following locations:
+Table 1: Merlin Containers
+
+| Container name | Container location | Functionality |
+|----------------|--------------------|---------------|
+| merlin-hugectr | https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-hugectr | Merlin and HugeCTR |
+| merlin-pytorch | https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-pytorch | Merlin and PyTorch |
+| merlin-hugectr | https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-tensorflow | Merlin and TensorFlow |
+
+## Examples and Tutorials
+
+We provide a collection of examples, use cases, and tutorials for [NVTabular](https://github.com/NVIDIA/NVTabular/tree/main/examples) and [HugeCTR](https://github.com/NVIDIA/HugeCTR/tree/master/notebooks) as Jupyter notebooks in our repository. These Jupyter notebooks are based on the following datasets:
+- MovieLens
+- Outbrain Click Prediction
+- Criteo Click Ads Prediction
+- RecSys2020 Competition Hosted by Twitter
+- Rossmann Sales Prediction
+  With the example notebooks we cover the following:
+- Preprocessing and feature engineering with NVTabular
+- Advanced workflows with NVTabular
+- Accelerated dataloaders for TensorFlow and PyTorch
+- Scaling to multi-GPU and multi nodes systems
+- Integrating NVTabular with HugeCTR
+- Deploying to inference with Triton
+
+For more sample models and their end-to-end instructions for HugeCTR visit the link: https://github.com/NVIDIA/HugeCTR/tree/master/samples
+
+## Learn More
+
+If you are interested in learning more about how NVTabular works under the hood, we have API documentation that outlines in detail the specifics of the calls available within the library.
+The following are the suggested readings for those who want to learn more about HugeCTR.
+
+HugeCTR User Guide: https://github.com/NVIDIA/HugeCTR/blob/master/docs/hugectr_user_guide.md
+
+Questions and Answers: https://github.com/NVIDIA/HugeCTR/blob/master/docs/QAList.md
+
+Sample models and their end-to-end instructions: https://github.com/NVIDIA/HugeCTR/tree/master/samples
+
+NVIDIA Developer Site: https://developer.nvidia.com/nvidia-merlin#getstarted
+
+NVIDIA Developer Blog: https://medium.com/nvidia-merlin
+
+## Contributing
```

Comment:
My $0.02 is to remove this section.  Motivated folks will find that information when they browse the repository.

Should we steal the idea of adding a "## Security CVEs" heading like these folks?
<https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow>